test/functional/test_framework/mininode.py

```diff
@@ -472,7 +476,7 @@ def test_function():
 
     def wait_for_verack(self, timeout=60):
         def test_function():
-            return self.message_count["verack"]
+            return "verack" in self.last_message
```

Comment:
why this change? I think waiting for `self.message_count["verack"]` to be non-zero matches exactly what we want the function to do.